### ISSUE TITLE
Article listing extra pattern design

### ISSUE THREAD

**Issue by tylersticka** [cloudfour/cloudfour.com-patterns#1089]
In our current design, we display some additional links to the right of summaries in [the Articles section](https://cloudfour.com/thinks/):

![Screenshot_2020-12-15 Cloud Four](https://user-images.githubusercontent.com/69633/102254176-2bd43280-3ebd-11eb-89ec-a668a5eb2ce8.png)

## Considerations

- Our new `o-deck` object feels best when it's full-width, so this should no longer be a sidebar but a full-width element.
  - Consider the possibilities that this offers: Could our "Essentials" be more compelling with excerpts, for example?
- In the future, this might accommodate a search field (which @megnotarte will be very happy about)?

**Comment by tylersticka** [comment]
@dromo77 Can you summarize what the state of this is? If it isn't actively in progress, can you un-assign yourself and move it to "Up Next"? Thank you!

**Comment by dromo77** [comment]
This work is still in the design phase. I created several mockups of the articles page, experimenting with adding sections based on topic:
![sections-lg](https://user-images.githubusercontent.com/42841342/118326817-594bf900-b4ba-11eb-947e-aa559cbf4599.png)
![sections-sm](https://user-images.githubusercontent.com/42841342/118326824-5b15bc80-b4ba-11eb-8515-cb05c775b68d.png)
![topics-sm](https://user-images.githubusercontent.com/42841342/118327204-7da7d580-b4ba-11eb-8731-7f5656eada24.png)

Feedback and other examples can be found in this [Slack thread](https://cloudfour.slack.com/archives/C04AEEVK2/p1620338008002900).

**Comment by tylersticka** [comment]
This was moved to "Done" so I'm going to close the issue.
